installing/installing_azure/enabling-user-managed-encryption-azure.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="enabling-user-managed-encryption-azure"]
+= Enabling user-managed encryption for Azure
+include::_attributes/common-attributes.adoc[]
+:context: enabling-user-managed-encryption-azure
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster with a user-managed encryption key in Azure. To enable this feature, you can prepare an Azure `DiskEncryptionSet` before installation, modify the `install-config.yaml` file, and then complete the installation.
+
+include::modules/installation-azure-preparing-diskencryptionsets.adoc[leveloffset=+1]
+
+include::modules/installation-azure-day2-operations-diskencryptionsets.adoc[leveloffset=+1]
+
+[id="enabling-disk-encrytpion-additional-resources"]
+== Additional resources
+
+* link:https://learn.microsoft.com/en-us/azure/virtual-machines/disks-enable-host-based-encryption-portal?tabs=azure-cli#prerequisites[Use the Azure portal to enable end-to-end encryption using encryption at host]
```

Comment:
OK, so this link goes to the official Azure docs. Please update the link as follows: 

```suggestion
* link:https://learn.microsoft.com/en-us/azure/virtual-machines/disks-enable-host-based-encryption-portal?tabs=azure-cli#prerequisites[Use the {azure-short} portal to enable end-to-end encryption using encryption at host] ({azure-full} documentation)
```

Now a customer will have some notice that they are leaving our docs when they click this link.